core/cmd/local_client_test.go

```diff
@@ -137,7 +137,7 @@ LEASE_LOCK_REFRESH_INTERVAL: 1s
 FLAGS_CONTRACT_ADDRESS: 
 LINK_CONTRACT_ADDRESS: 
 LOG_FILE_DIR: %[1]s
-LOG_LEVEL: debug
+LOG_LEVEL: 
```

Comment:
Should this be "info" or is that set somewhere else?

Reply:
Zap maps empty to info, so this will work fine, but it might not hurt to make it explicit.

Reply:
This is actually the one exception we need to override to align with CI, so switching back.